Atomic/syscontainers.py

```diff
@@ -786,6 +821,7 @@ def extract_system_container(self, img, destination):
         repo = self._get_ostree_repo()
         if not repo:
             return False
+        self.args.remote = None
```

Comment:
I would store this information in the `info` file as well, it could be used by `update_system_container` without checking for the existence of "roots"